.gitignore

```diff
@@ -3,3 +3,4 @@
 *.jl.cov
 *.jl.*.cov
 docs/build
+Manifest.toml
```

Comment:
This should start with a `/`; otherwise it ignores files named `Manifest.toml` anywhere in the entire repo.